### DISCUSSION TITLE
Sunbird - Update branding colour in classic theme

### DISCUSSION THREAD

**Discussion by Ekta29yadav** [Sunbird-Ed/Community#735] [Enhancement request]
Hi team,
Hope you are doing well!

We have a requirement from client to change the classic theme as per branding colour.
Currently we have observed that themes are coming from @project-sunbird/sb-themes which we don't want to customise and maintain.
Is there any way to change at frontend ED portal for this?

**Please help us to achieve this custom theme changes.**

We appreciate any insights. Thank you!
@sri-pusuluri , @vinukumar-vs ,  @rajeevsathish , @komalm, @manishakapadnis, @paritshivani

**Comment by Ekta29yadav** [comment]
@sri-pusuluri Any update on this?
 @vinukumar-vs , @rajeevsathish , @komalm, @manishakapadnis, @paritshivani

**Comment by Ekta29yadav** [comment]
Hi team,

Hope you are doing well!

We have a requirement to display only the classic theme after login by default, instead of the joyful theme. Is there any way to remove the joyful theme so that only the classic theme is available?

Additionally, our client has requested to modify the classic theme according to their branding colors. Currently, we have observed that the themes are coming from @project-sunbird/sb-themes, which we prefer not to customize and maintain.

Is there any way to implement these changes directly in the frontend ED portal?
We appreciate any insights. Thank you!
@sri-pusuluri , @vinukumar-vs , @rajeevsathish , @komalm, @manishakapadnis, @paritshivani

**Comment by HarishGangula** [comment]
@Ekta29yadav  classic theme is deprecated some functionalities may break, Please use joyful theme, update the theme configs and variables in sb-themes as per your need, publish that as package and update the new package as dependency in portal to see the changes.

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
